android/src/test/java/vn/luongvo/kmm/survey/android/ui/screens/login/LoginViewModelTest.kt

```diff
@@ -0,0 +1,105 @@
+package vn.luongvo.kmm.survey.android.ui.screens.login
+
+import app.cash.turbine.test
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.*
+import org.junit.*
+import vn.luongvo.kmm.survey.android.test.CoroutineTestRule
+import vn.luongvo.kmm.survey.android.test.Fake.token
+import vn.luongvo.kmm.survey.android.ui.navigation.AppDestination
+import vn.luongvo.kmm.survey.domain.usecase.IsLoggedInUseCase
+import vn.luongvo.kmm.survey.domain.usecase.LogInUseCase
+
+@ExperimentalCoroutinesApi
+class LoginViewModelTest {
+
+    @get:Rule
+    val coroutinesRule = CoroutineTestRule()
+
+    private val mockIsLoggedInUseCase: IsLoggedInUseCase = mockk()
+    private val mockLogInUseCase: LogInUseCase = mockk()
+
+    private lateinit var viewModel: LoginViewModel
+
+    @Before
+    fun setUp() {
+        every { mockIsLoggedInUseCase() } returns flowOf(false)
+        every { mockLogInUseCase(any(), any()) } returns flowOf(token)
+
+        viewModel = LoginViewModel(
+            mockIsLoggedInUseCase,
+            mockLogInUseCase
+        )
+    }
+
+    @Test
+    fun `when calling init successfully with false, it shows login page`() = runTest {
```

Comment:
We used `-` in this [PR](https://github.com/luongvo/kmm-survey/pull/51) but `,` in this PR? Is it because on Android target, it can read the `,`?

Reply:
@blyscuit Exactly, on the Android module & library, we can use more special chars with backtick func name (please check out more examples [here](https://github.com/nimblehq/jetpack-compose-crypto/blob/develop/app/src/test/java/co/nimblehq/compose/crypto/ui/screens/home/HomeViewModelTest.kt)). Unfortunately, with KMM library module, it has more limitations to back to ` - ` instead of `, ` 😄